dds-suite.repos

```diff
@@ -31,3 +31,19 @@ repositories:
         type: git
         url: https://github.com/eProsima/ShapesDemo.git
         version: v2.4.1
+    Micro-CDR:
+        type: git
+        url: https://github.com/eProsima/Micro-CDR.git
+        version: v2.0.0
+    Micro-XRCE-DDS-Client:
+        type: git
+        url: https://github.com/eProsima/Micro-XRCE-DDS-Client.git
+        version: v2.1.0
+    Micro-XRCE-DDS-Agent:
+        type: git
+        url: https://github.com/eProsima/Micro-XRCE-DDS-Agent
+        version: v2.1.0
+    Micro-XRCE-DDS-Gen:
+        type: git
+        url: https://github.com/eProsima/Micro-XRCE-DDS-Gen
+        version: v1.1.0
```

Comment:
Please add a new line at the end of the file
```suggestion
        version: v1.1.0

```

Reply:
Done